internal/restore/pvc_action.go

```diff
@@ -31,10 +31,14 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 
 	"github.com/vmware-tanzu/velero-plugin-for-csi/internal/util"
-	velerov1api "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
 	"github.com/vmware-tanzu/velero/pkg/plugin/velero"
 )
 
+const (
+	AnnBindCompleted     = "pv.kubernetes.io/bind-completed"
+	AnnBoundByController = "pv.kubernetes.io/bound-by-controller"
```

Comment:
Looking in my current OpenShift cluster, I see that there are actually two other cluster-generated annotations on my PVCs. I don't know that it matters all that much whether they are removed or retained, but to minimize the change from this PR (before the PR, everything but velero annotations is removed, after the PR, only annotations in this list are removed), it might make sense to add these other two to the remove list as well:
```
               "volume.beta.kubernetes.io/storage-provisioner"
               "volume.kubernetes.io/selected-node"
```

Reply:
Commit updated.